packages/payment-integration-test-framework/src/fixture/pageObject/playwright/senstiveDataConfig.ts

```diff
@@ -1,16 +1,21 @@
 // Sensitive headers would be masked in a HAR file.
 export const sensitiveHeaders = [
+    'accept',
+    'alt-svc',
     'authorization',
+    'cf-cache-status',
+    'cf-ray',
     'cookie',
+    'nel',
+    'referer',
+    'report-to',
```

Comment:
Many new `cloudflare`-related headers appeared in the HAR file, which, in my opinion, revealed far too many tech details. 

Therefore, I updated the `sensitiveHeaders` config.